app/assets/stylesheets/static_pages.scss

```diff
@@ -4,45 +4,49 @@
 
 @import "ashby_settings.scss";
 
-.banner_text {
-  font-style: bold;
-  padding-bottom: 150px;
-}
-
-.banner_text h1, h4 {
-  color: white;
-}
-
-.columns p {
-  color: $ashby-gray;
-}
-
 .landing {
   text-align: center;
   background: image-url("bg_main.jpg") transparent no-repeat;
   background-size: contain;
-  background-position: 0% -50%;
-  padding: 200px;
-}
+  background-position: 100% -50%;
```

Comment:
Pretty certain I wrote this in the super hacky push before GM. What I propose is that we can merge this branch in, but I'll make my next focus to clean up the styles for responsiveness. What say you @lilshim?